lib/router.js

```diff
@@ -1857,6 +1857,8 @@ router.get('/tencent/pvp/newsindex/:type', lazyloadRouteHandler('./routes/tencen
 
 // 《明日方舟》游戏
 router.get('/arknights/news', lazyloadRouteHandler('./routes/arknights/news'));
+// 《明日方舟》游戏内公告
+router.get('/arknights/announce/:platform?/:group?', lazyloadRouteHandler('./routes/arknights/announce'));
```

Comment:
New route should follow [v2 standard](https://docs.rsshub.app/joinus/script-standard.html)